lib/TestRunner.lua

```diff
@@ -110,7 +144,7 @@ function TestRunner.runPlanNode(session, planNode, tryStack, noXpcall)
 			end
 		elseif childPlanNode.type == TestEnum.NodeType.Describe or childPlanNode.type == TestEnum.NodeType.Try then
 			if childPlanNode.type == TestEnum.NodeType.Try then tryStack:push({isOk = true, failedNode = nil}) end
-			TestRunner.runPlanNode(session, childPlanNode, tryStack, childPlanNode.HACK_NO_XPCALL)
+			TestRunner.runPlanNode(session, childPlanNode, tryStack, lifecycleHooks, childPlanNode.HACK_NO_XPCALL)
```

Comment:
This function's starting to get a lot of arguments. We're right at the threshold where refactoring it to take a table of arguments might be a good idea, but we can leave that out of this PR.